hack/build-go.sh

```diff
@@ -41,6 +41,8 @@ if [ "$GO111MODULE" == "off" ]; then
 	export GOBIN=${PWD}/bin
 	export GOPATH=${PWD}/gopath
 	go build -o ${PWD}/bin/multus -tags no_openssl -ldflags "${LDFLAGS}" "$@" ${REPO_PATH}/cmd
+	go build -o ${PWD}/bin/multus-daemon -tags no_openssl -ldflags "${LDFLAGS}" "$@" ${REPO_PATH}/cmd/thick
```

Comment:
How about to change the file layout as following?
- cmd/multus
- cmd/multus-daemon (I feel that 'multus-daemon' is for thick plugin server, so you may change the name of binary because it might be simple loop for thin plugin, current multus)
- cmd/generate-kubeconfig

Reply:
I'm OK with that; I originally named it because using the thick plugin option sounded like a certainty at the time. 

I'll try to come up with something better: `multus-controller` possibly, or `pod-watcher`, `pod-monitor` , etc. 

I think I prefer `multus-controller` since it sounds less detailed (if in the future in starts watching some other resource, the binary name becomes misleading).

Reply:
@s1061123 does this new directory structure make sense to you ?